script/bash_include

```diff
@@ -55,7 +55,7 @@ trap clean_log EXIT
 # Grab constant from rails configuration (consts.rb and consts-site.rb).
 #   path=$( rails_constant MO.local_image_files )
 function rails_constant {
-  $app_root/script/config.rb $*
+  rails runner $app_root/script/config.rb $*
```

Comment:
I think it's totally cool that "rails runner" does exactly what my crappy config.rb script did! :)